DOCUMENTATION.md

```diff
@@ -157,78 +187,104 @@ PQQDAgNIADBFAiEA+vajlz7HDZ7x6KKi/uMlrwCePEcchZRYJPc/6kPyYogCIFSy
 etQ54MyIOWtwYlxG+blnxT4PWCgas5rPiaK6VP/Z
 -----END CERTIFICATE-----
 ```
-- `discovery_address`: **string**  this is the address of the discovery server to use in order to find back the address of the handler to which the application in registered. It's composed by the hostname and the port number, example: `discovery.thethings.network:1900`.
-- `path_to_key=`: **string**  this is the absolute path to the file which contains the key from which the token you wish to use, is signed.
-The constructor returns an **ApplicationClient** object set up with the application informations, ready to get the application registered on The Things Network.
+
+* `discovery_address`: **string** this is the address of the discovery server to use in order to find back the address of the handler to which the application in registered, in the `{hostname}:{port}` format. Example: `discovery.thethings.network:1900`.
+* `path_to_key=`: **string** this is the absolute path to the file which contains the key from which the token you wish to use, is signed.
```

Comment:
What's exactly the purpose of this? In what situation do you see a use for this?

Reply:
The node sdk provides the possibility to connect to an application with a token signed with a key, the path_to_key argument is to provide the path to the file used to check if the string passed is indeed a token or an app access key.
I'm not sure it is very often used, maybe I could remove it?

Reply:
@romeovs what use case does this answer to in the Node SDK?

Reply:
It does not have an equivalent. And frankly I find it confusing. I don't see what it is or how it would be used.

Reply:
Okay then I'm removing it